src/services/config/config.service.ts

```diff
@@ -31,3 +31,5 @@ export default class ConfigService {
     return configs[key];
   }
 }
+
+export const configService = new ConfigService();
```

Comment:
I don't think we ever need more than one instance of `ConfigService` right? so I just init it once here and export it for usage.